packages/edit-post/src/index.js

```diff
@@ -161,7 +162,7 @@ export function reinitializeEditor() {
 }
 
 export { default as PluginBlockSettingsMenuItem } from './components/block-settings-menu/plugin-block-settings-menu-item';
-export { default as PluginDocumentSettingPanel } from './components/sidebar/plugin-document-setting-panel';
+export { PluginDocumentSettingPanel };
```

Comment:
Should we wrap this in a deprecated call to let extenders know about the change so they can change their implementations? 

I don't think we would need a removal timeframe for now. But from what I can tell here it doesn't make sense to use the version exported through there going forward. 

It's definitely important that it is here though to not break anything :)

Reply:
Yes, I wondering about it. Should we deprecate now or give it some time (maybe move more slots and check feedback)